docs/troubleshooting.md

```diff
@@ -0,0 +1,129 @@
+# Troubleshooting Marketplace Operator
+
+This document lists out some common failures related to the Marketplace Operator a user might encounter, along with ways to troubleshoot the failures. If you encouter a failure related to the Marketplace Operator that is not listed in this document, but should be, please open an issue or a PR to have the failure appended to this document. 
+
+The troubleshooting steps listed here are for Marketplace resources like OperatorSource and CatalogSourceConfig. To troubleshoot [Operator-lifecycle-manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager) defined resources like ClusterServiceVersion, InstallPlan, Subscription etc, refer to the [OLM troubleshooting guide](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/debugging.md).
+
+Note that all examples in this doc are are meant to be ran against an Openshift cluster. All examples (except for the ones involving the UI) should work with `kubectl` and with the appropriate namespaces substituted in them.   
+
+Table of contents
+===================
+
+1. [No packages show up in the UI (No OperatorHub Items Found)](#no-packages-show-up-in-the-ui-(no-operatorhub-items-found))
+2. [OperatorSource failing to download from datastore](#operatorSource-failing-to-download-from-datastore)
+3. [OperatorSource `MESSAGE` reads unknown error (status 404)](#operatorsource-`message`-reads-unknown-error-(status-404)) 
+4. [CatalogSourceConfig stuck in `Configuring` phase](#catalogsourceconfig-stuck-in-configuring-phase)
+
+
+## No packages show up in the UI (No OperatorHub Items Found)
+
+When you install an Openshift cluster, the embedded Marketplace (OperatorHub) comes with a default list of operators that can be installed onto the cluster. A number of resources need to be investigated if however you see a message like this: 
```

Comment:
`OpenShift`

Reply:
We should not mention `embedded Marketplace` when referring to the UI as that will confuse the reader who does not know the history. Best to stick with OperatorHub.